### DISCUSSION TITLE
【职场】2025年 - 企业招聘

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#216] [Show and tell]
## 欢迎大家踊跃留言内推招聘信息

> 今年的行情的确太差了，有些兄弟都没找到工作
> 时代的一粒沙，落在每个人头上都是一座山，希望好兄弟相互帮忙

- 格式： 联系方式 + 地域 + 岗位 + 职业要求（或链接） 
- 有些内容不方便的，可以留个人邮箱什么的

---

短短这2周，群里面有3个兄弟 （三个城市）出现被公司强制辞退。。
大家后续有内推的，发出来，最近群里面被辞退的人明显在增加！

需要内推找人的 at 片刻（微信：529815144），我会帮忙转发，帮助被辞退的找工作！
需要律师的，片刻也可以帮忙推荐，你可以咨询 。。。

---

<img width="311" alt="image" src="https://github.com/apachecn/home/assets/9199175/21e99b7c-6617-46a6-ba90-b2fa5c123ed3">

**Comment by jiangzhonglian** [comment]
## 滴滴智能平台事业群-北京
- 联系方式：xiewenzhen@didiglobal.com

### 语音算法工程师

> 职位描述

1）工作包括不局限于：声纹识别、属性识别、声音分类识别、语音分离、语音识别等算法；
2）跟踪业界最新的语音算法和技术，并应用到业务中；
3）与团队成员合作，按时完成项目研发、测试和上线。

> 任职要求

1）计算机科学等相关专业本科或以上学历；
2）有语音识别技术工作经验，熟悉端到端识别技术；
3）熟悉Kaldi、ESPNet等各种开源语音工具；
4）在语音技术领域发表过顶会论文或者比赛获奖的优先。


### 视觉算法工程师

> 职位描述

1）工作包括不局限于：目标检测、跟踪、定位、深度距离估计、视频理解等算法
2）团队成员合作，按时完成研发工作并进行算法落地。

> 任职要求

1）计算机科学等相关专业本科或以上学历；
2）有计算机视觉背景工作经验，熟悉ffmpeg、opencv、pytorch等工具；
3）有推理加速和嵌入式硬件优化经验，熟悉NCNN、TensorRT等加速工具优先；
4）在计算机视觉领域发表过顶会论文的优先。

**Comment by jiangzhonglian** [comment]
## 大疆内推

- https://we.dji.com/zh-CN/campus/position?source=wechat&project=intern&trigger=forward_message
- 内推码：

```
RVH4CKGR
AATVSQNW
WLJB8PQB
UX4V6TZX
VUWPAJTC
BUDVY5L4
NRDGXVX8
L8JPJMY8
63AZCKP9
TBNX8NT4
```

**Comment by jiangzhonglian** [comment]
## 百度 - 北京电通创意广场

- xiaoyupopper（微信）/ 17301344173（电话）

### 前端

> 工作职责

1、负责前端页面构建，各类交互设计与实现；
2、前端样式和组件开发的模块开发；
3、精通前端工程化构建工具，至少有一个大型项目开发经验；
4、配合后台开发人员完成项目；

> 岗位要求

1、6年及以上的前端开发经验；或4年以上百度前端开发经验
2、了解W3C标准，熟练使用HTML、CSS、JavaScript、Ajax、DOM等前端技术；
3、了解WEB标准化、模块化、性能优化方法，了解可用性、可访问性和安全性；
4、精通vue.js架构，至少3年vue.js开发经验，并有一定的组件开发能力；
5、有浏览器兼容性及性能优化经验者优先；
6、精通CSS3者优先；
7、有良好的代码编写习惯，有自主学习能力，有良好的团队合作意识及积极的心态。

### Java 

> 工作职责

1、负责后端程序的架构设计与开发，并与Web前端沟通协作;
2、负责后台数据库架构设计，至少熟悉一个关系型数据库 (如MySQL、SQL Server) 以及相关的数据库调优技巧;
3、按照需求对产品后端程序功能进行修改和升级;负责代码的优化和维护，保障运行效率;
4、跟进后端技术发展趋势，及时用更高级的技术来提高后端程序的运行效率。

> 任职要求

1、6年及以上开发经验，具有Web类相关开发经验2年以上
2、对Java面向对象软件结构有深入的理解以及很强的应用能力;
3、能够熟练使用一门或几门以下的技术进行软件开发:C/C++/Java、Linux/Unix Shell编程Perl/Python/PHP、网络技术及相应的开发JavaScript/Html/Ajax、MySq/Oracle及相关数据库技术及常用优化方案
4、对数据库基本理论和内部实现机制有比较深刻的理解，有千万量级数据库设计及优化能力;
5、熟练使用Linux或Unix;熟悉HTTP协议，XML/HTML/CSS/Javascript等Web开发相关技
6、熟练使用mysql/oracle等数据库进行后台开发


### 大数据开发

> 工作职责

参与团队大数据开发工作，包括但不限于了解业务数仓需求，开发spark算子，运维并调优spark运行性能

> 任职要求

1、5年及以上开发经验；熟练掌握大数据技术知识
2、熟悉并使用过spark，hadoop，hdfs，hive，flink等大数据相关的组件框架
3、精通SQL，及NOSQL
4、有良好的业务理解能力，能够思考数仓内各表，各字段的内在业务联系

### Python

> 工作职责

辅助策略模型的部署、服务包装、调用及调试

> 任职要求

1、  5年及以上开发经验；精通python及python常用工具库
2、  熟练使用web框架，如flask，具有良好的后端技术知识及项目经验
3、  对机器学习、深度学习有熟悉，使用过numpy，sklearn，pytorch 等加分

**Comment by jiangzhonglian** [comment]
## 首钢 - 石景山

- xiaoyupopper（微信）/ 17301344173（电话）

### Java

> 技能要求

1、Java基础扎实，深入理解Spring、Spring boot等框架的设计原理及实现；
2、熟练使用Mybatis，Oracle，Redis，RabbitMQ，ELK（Elesticsearch、logstash、kibana）；
3、熟练掌握前端基础知识，包括但不限于html、css、js以及相关框架；
4、熟练使用Vue.js框架，熟练使用Element UI。

> 岗位职责

1、根据产品功能设计，编码实现各模块功能，确保开发质量；
2、负责程序开发工作，有良好的编码习惯；
3、研究项目技术细节，编写相应的说明书；
4、参与程序代码单元测试、上线工作。

**Comment by jiangzhonglian** [comment]
## 南京一上市公司招聘软件

- 联系邮箱：gxy_dl@163.com


### 项目经理

负责项目内部研发流程管理

> 要求

1. 三年及以上项目流程管理经验，计算机相关专业
2. 熟悉敏捷或IPD研发流程，对精益有一定理解和实践
3. 善于思考问题，有较强逻辑思维和业务理解能力
4. 精通jira、confluence、缺陷管理、版本管理等工具
5. 有较强沟通能力，善于处理冲突，协调资源。

> 有以下经验者优先

1. 有互联网项目管理经验，有0-1项目体系化建设经验
2. 有产研效能度量体系建设和提升经验
3. 对项目改进有独特见解，有持续改进实践经验
4. 有质量管理，风险管理实践经验
5. 有CI，DevOps相关实践经验。

**Comment by jiangzhonglian** [comment]
> 理想汽车社招内推码: 8YTDNKZ 

* [投递链接](https://li.jobs.feishu.cn/referral/m/position?token=MzsxNjg5ODM4NjMzNDA3OzcxMTQ4MzEzNTczMDIwMTM5ODA7MA)

**Comment by xiaoxiezzy** [comment]
### 广州-埃森哲（~08/31）

・以上机会欢迎自荐或推荐，咨询微信：zzy_211025
・简历邮箱地址：yonghengzzy@sina.cn

**_Java工程师，职级10-8 ，英语流利_**

> 职位描述

1. 负责Java后端程序功能实现，API设计和编写，以及测试
2. 负责Web前端设计实现，编写前端逻辑代码，与后端整合
3. 需求方沟通以及讨论需求
4. 与设计师合作参与UI设计

> 主要技能

Java Enterprise Edition

> 任职要求

1. 熟悉Java，SpringBoot，微服务架构；
2. 有两年以上Java项目经验，熟悉项目开发流程；
3. 具备较强的粤语及英语沟通能力者优先。

**_UIUX设计师，职级9-8 ，英语流利_**

> Priority Skill

Architecture Design

> Responsibilities

•  Will focus more on business to employee demands
•  Ongoing work with Product Owner, stakeholders and developers on different projects (not project base)
•  Conduct UX research, e.g. benchmarking, market research with adjacent industry
•  Perform user testing, which brings insights and help address pain points from users
•  Perform usability test for design validation that ensures we are building the right product with continuous improvements
•  Implement and deliver UIUX design for projects

> Qualifications and Skills

•    An updated portfolio or samples of work demonstrating experience creating user-centered design solutions.
•    Proficiency in a variety of design and prototyping tools including but not limited to: Figma, Sketch, Adobe design tools.
•    Minimum 5 years of experience in UX/UI design, Interaction design or equivalent experience.
•    Must have strong interaction and visual design skills with the ability to design quality graphical user interfaces.
•    Experience conducting user research, including contextual inquiry and usability & accessibility testing.
•    Experience to organize and host online design workshop to support requirement gathering.
•    Excellent oral and written communication skills in English.

> Education and Experience

•    Bachelor's degree in Design / Human-Computer Interaction (HCI), Psychology, or equivalent experience
•    Have an understanding of Agile methodologies and SCRUM

**_前端工程师 (node.js/angular), 职级10，粤语流利_**
**_前端工程师 (react/node.js), 职级9-7，英语流利_**

> 岗位职责

1、使用node.js/react-native/angular进行应用研发，打包及发布
2、根据开发需求完成相关代码编写
3、将业务需求转化为软件应用程序和模型
4、开发完成时执行单元测试，提供UAT和后期实施支持/生产支持，遵循质量流程，确保达到质量目标

> 主要技能

Node.js

> 任职要求

1、5年以上软件开发经验，其中2年及以上node.js/react-native/angular进行相关应用开发项目经验
2、理解并掌握原生JavaScript核心概念和原理
3、了解应用程序开发过程的所有阶段，从设计、开发、测试，到实现和培训，有application support的知识和经验；
4、逻辑思维能力强，具备良好的合作态度及团队精神；
5、自我驱动意识及学习能力强；
6、英语流利（加分项：粤语熟练）

**Comment by jiangzhonglian** [comment]
## 成都一好兄弟的公司内推

- 微信：t771915103
- 联系邮箱：771915103@qq.com 
- 更新时间：2023-09-01

### 高级JAVA开发工程师（P7为主）

> 岗位职责

1. 负责公司物联网相关业务设计，开发和优化
2. 参与项目的设计、方案评审和技术难点攻关
3. 深入理解业务，围绕业务的发展、产品规划等做技术规划，并且能够驱动落地 ；

> 任职资格

1. 计算机相关专业，本科及以上学历；3年以上java及分布式系统开发经验，1年以上系统设计经验；
2. 熟悉Java及Java EE体系结构，理解Redis，MySQL等的基本原理，并熟练恰当的运用到业务场景中；
3. 精通Flink、Storm等实时计算框架中的至少一项，有实时计算平台的开发和调优经验，有海量数据处理经验者优先；
4. 具备扎实的计算机专业基础，包括算法和数据结构、操作系统、计算机体系结构、计算机网络、数据库等；
5. 具备良好的沟通表达能力、极强的责任心和团队合作精神，乐于和热于技术钻研；
6. 对大数据场景消息中间件kafka、rocketmq原理了解者优先；
7. 高并发分布式计算或海量数据应用经验者优先，具有物联网(IoT)行业经验者优先。

**Comment by Abel-Huang** [comment]
### 美团-北京朝阳

- 微信/电话: 18861823615
- 邮箱：497720975@qq.com
- 更新时间：2023-10-11

#### 职位
供应链研发工程师(Java)

#### 岗位要求
1. 计算机或相关专业，本科，工作年限三年以上
2. 熟悉Java，熟悉JVM，SpringBoot等Java生态
3. 熟练使用MySQL，Redis，Kafka等中间件
4. 对供应链物流有一定的认识

#### 备注
本岗位可直接联系，其他岗位可以使用内推码：vIkTZVb

**Comment by jiangzhonglian** [comment]
### 广州海珠 - 群友内推

- 微信号：devlearning
- 更新时间：2023-10-11

#### 职位

前端工程师（一个）

#### 岗位要求

15-25k 一年15-18薪 公积金顶格双边 12%
双休，真965，最好是 react 三年以上。
节假日前一天，下午四点就下班。
对了，每个月还有650的餐补。
有想法的可以私我。

**Comment by jiangzhonglian** [comment]
## 南京(三维天地) - 群友超哥内推

* 联系电话： 19150151392
* 更新时间：2023-10-11

职位：供应链研发工程师(Java)

岗位要求：

1.计算机或相关专业，本科，工作年限三年以上
2. 熟悉Java，熟悉JVM， SpringBoot等Java生态
3. 熟练使用My SQL，Redis, Kafka等中间件

**Comment by jiangzhonglian** [comment]
## 合十思维（北京）科技有限公司 - 好兄弟内推（建议直接电话私聊）

* 联系方式：王家航-13084879260
* 北京市昌平区回龙观腾讯众创空间A512
* 薪资待定（价格可以细聊）
* 更新时间：2023-10-23

> 语音算法工程师

要做的事情就是：

- 语音识别（普通话转中文文本）
- 语音合成（这块可能涉及到语音的情感，声音克隆等等）
- 用户唱的歌和歌曲匹配

> 这个是招聘的内容 - 扫码就可以看到（比较刻板，你想看也行）

<img src=https://github.com/apachecn/.github/assets/9199175/0dde5000-93e2-4db8-8b27-d864aef2acf4 width=20% />

<img src=https://github.com/apachecn/.github/assets/9199175/f4d75dae-d8ae-469f-96ca-4d3b6d806efd width=40% />

**Reply by jiangzhonglian** [reply]
预计50万包年，可以细聊（具体还是以聊的为准）

**Comment by jiangzhonglian** [comment]
## 滴滴出行 - 好兄弟内推(速来！)

* 联系方式： ssozh1999 （微信）
* 更新时间：2023-12-04

工作地点：北京
职位类别：技术
所属部门：货运事业部

### 高级JAVA研发工程师 (J230920001)

> 职位描述

1、深度参与用户运营技术体系的设计和研发工作，包括用户增长、用户侧核心交易、渠道管理、CRM体系等多个方向，并负责其稳定性工作；
2、参与项目的系统分析，设计工作，承担核心功能，公共核心架构模块的代码编写；
3、解决各种疑难杂症，系统优化，并且完成产品、平台和组件的维护和沉淀。

> 任职要求

1、本科以上学历，计算机软件专业，2年以上java开发经验；
2、java基础扎实，熟悉集合、io/nio、多线程等基础知识，熟悉分布式、缓存、消息队列等主流技术；
3、熟练掌握spring、mybatis等开发框架，熟悉主流开发流程，熟悉各种常用的设计模式；
4、熟悉软件建模，有一定的系统架构设计能力；
5、具备较强的业务sense，有深入了解业务的意愿；
6、具备较强的抗压能力和良好的沟通技巧，具有优秀的团队合作精神，善于学习和深度思考，做事积极主动有成长意愿。

**Comment by jiangzhonglian** [comment]
# 社招 | 大疆机器学习团队热招职位

* 更新时间：2024-02-23
* 工作地址：深圳

> 内推地址：https://b.xiumi.us/board/v5/4vEE7/514431911

大疆机器学习部，划分走一批人到产品部，因此空缺出一批HC，3月底就需要招聘完成，因此招聘流程大大简化。直接内推到各个小组，并开始面试。业务主要支持飞机和手持产品的AI功能，稳定性非常高，薪资业内TOP，岗位匹配度高的小伙伴面试成功率极高，学历会适当放宽。

- 可以直接联系（微信也是手机）： 13020064554

**Comment by jiangzhonglian** [comment]
## 沈阳 - 全职 JAVA(8K)- 对接日本项目

* 联系方式：529815144 （微信）
* 更新时间：2025-06-25
* 需求人数：10人

### JAVA开发工程师

我们现在有批量对日Java需求，在沈阳，长期短期都有，学信网可查大专，

会日语/有对日项目的1-2年，不会日语的3年以上开发经验，愿意做对日项目的

**Comment by samsonJjiang** [comment]
### **宇树科技股份有限公司**

- 关于我们

宇树科技（Unitree）专注于消费级、行业级高性能通用足式/人形机器人及灵巧机械臂的自主研发、生产和销售，且高度重视自主研发和科技创新，全自研电机、减速器、控制器、激光雷达等机器人关键核心零部件和高性能感知及运动控制算法，整合机器人全产业链。在具身智能领域我们拥有完备的团队与理想的落地场景。


- 投递方式

直接发送到邮箱[hr_jzx@unitree.com](mailto:hr_jzx@unitree.com)，邮件备注[片刻]，感谢片刻大大


- 岗位介绍

1. AI算法工程师（VLA方向）
关键字：VLA、世界模型、具身智能
职责描述：
1.负责大模型(视觉或语言)的预训练开发和调优工作；
2.负责大模型在具体业务场景的适配和应用，如fine-tuning.prompt设计等；
3.负责大模型的推理优化和应用部署，如压缩、量化、推理加速等。
4.负责跟进领域最新进展，预研和评估大模型在机器人领域的应用，如Gato、RT-1、PaLM-E。
任职要求:
1.硕士及以上学历，985/211院校毕业优先。计算机相关专业，熟悉深度学习、计算机视觉、自然语言处理等领域的专业知识，具有3年以上工作经验；
2.熟悉Transformer、BERT、GPT系列、T5、PaLM等模型，对大模型的数据处理及训练技术有深入理解；
3.熟悉Pytorch/Tensorflow深度学习框架，至少掌握Python或C++其中一种编程语言；
4.熟悉常见的强化学习算法原理(如DQN、DDPG、A3C、PPO等)，有相关训练和开发经验者优先；
5.具有TensorRT、ONNX推理优化和部署经验者优先；
6.曾在AI、CV、NLP顶会(如：AAAI、CVPR、ACL等)发表论文者优先。


2. 项目经理
关键字：推动力、主动性、PASSION！！！
岗位职责：
1.寻找挖掘应用场景和客户需求，规划及细化产品的需求点和设计点；
2.规划及跟进产品研发周期，协调研发同事，跟踪产品开发进度，完成产品的开发、测试、量产、发布等相关工作，并撰写项目相关报告；
3.协助研发/销售团队开发、交付和维护满足顾客需求的市场驱动型产品；
4.产品落地相关认证调查及实施；
5.积极开展对外合作，推进合作、外包等项目的进展。
任职要求：
1.统招本科及以上学历，自动控制、机电等相关专业；
2.2年以上产品研发岗位经验，2年以上新产品的管理经验，有成功量产及销售的产品经验，机器人、机械臂方向优先，有机电类产品工作背景经验优先；
3.管理客户期望，能够协调产品需求点和技术点之间的矛盾，遵守严格的工程开发流程，协调各部门高效推进产品进度；
4.具备卓越的执行能力，具有优秀的沟通、协调、抗压能力强，能够有效地组织和推进跨职能、跨团队的合作项目；
5.具备较强的逻辑思维能力；具备对并行项目的整体把控能力；具备良好的语言组织能力，能够理解技术要点，并形成完整的报告材料。

**Comment by jiangzhonglian** [comment]
## 鸿蒙 - 开发(中高级+初级) + 测试

* 联系方式：529815144 （微信）
* 更新时间：2025-07-26
* 需求人数：20人+

### 开发：(薪资私聊)（开发需要鸿蒙证书）

1.  本科工作经验3年以上，专科工作经验5年以上 （中高级）
    本科工作经验1年以上，专科工作经验2年以上 （初级）
2.  需要有1年实际的鸿蒙经验
3.  开发的JD，安卓/iOS/RN/前端/C++方向都行
4. 音视频、网络通信、H5、近场通信(NFC、蓝牙)经验优先
5. C++经验优先

### 测试：(薪资私聊)
1. 本科工作经验3年以上，专科工作经验5年以上
2. 熟悉鸿蒙系统原理，具备丰富的测试经验并掌握系统性测试方法
3. 1年以上APP端测试经验
4. 有电商、银行相关测试经验优先

北上广深杭州南京以及其他省会城市都可以招，但是要接受全国出差（base地以外才算出差，出差补贴很不错，感兴趣私聊）
